numpy/testing/noseclasses.py

```diff
@@ -298,19 +300,14 @@ def configure(self, options, config):
                                   if p.name != self.to_unplug]
 
 
-class KnownFailureTest(Exception):
```

Comment:
This worries me. For instance, `KnownFailureTest` is used in many places in Scipy, and likely in other projects also.